src/utils/tokenInfo.ts

```diff
@@ -50,3 +52,13 @@ export const getFormattedAddress = (address?: string, fallback?: string): string
     return fallback || address || ''
   }
 }
+
+export const isTokenNative = (currency: Currency | WrappedTokenInfo | undefined, chainId: ChainId | undefined) => {
+  if (currency?.isNative) return true
+  // case multichain token
+  return chainId
+    ? nativeOnChain(chainId).wrapped.address === currency?.address &&
```

Comment:
you can use `WETH[chainId]` for shorter

Reply:
what is this @@
<img width="332" alt="image" src="https://user-images.githubusercontent.com/33005392/198213151-ad534ad7-9fe7-4eed-b990-d82f6f8c3bc1.png">